recipes/scherbelberg/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "scherbelberg" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scherbelberg-{{ version }}.tar.gz
+  sha256: a87fb41eb666a3a5d5b0c1aa2caaa2e10a646c382bb88a4373c997dd1dc2a708
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - "scherbelberg = scherbelberg._cli:cli"
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - python >=3.8
+    - click
+    - dask
+    - hcloud
+    - pyyaml
+    - typeguard
+
+test:
+  imports:
+    - scherbelberg
+    - scherbelberg._cli
+  commands:
+    - pip check
```

```suggestion
    - pip check
    - scherbelberg --help
```
Just to check the CLI.

---

Good idea.